packages/polyfills-loader/src/create-polyfills-data.js

```diff
@@ -0,0 +1,199 @@
+       */
+      test:
+        "'noModule' in HTMLScriptElement.prototype && " +
+        "(function () { try { Function('window.importShim = s => import(s);').call(); return true; } catch (_) { return false } })()",
+      path: require.resolve('dynamic-import-polyfill/dist/dynamic-import-polyfill.umd.js'),
+      initializer: "window.dynamicImportPolyfill.initialize({ importFunctionName: 'importShim' });",
+    });
+  }
+
+  if (polyfills.esModuleShims) {
+    addPolyfillConfig({
+      name: 'es-module-shims',
+      test: "'noModule' in HTMLScriptElement.prototype",
+      path: require.resolve('es-module-shims/dist/es-module-shims.min.js'),
+      fileType: fileTypes.MODULE,
+    });
+  }
+
+  if (polyfills.intersectionObserver) {
+    addPolyfillConfig({
+      name: 'intersection-observer',
+      test:
+        "!('IntersectionObserver' in window && 'IntersectionObserverEntry' in window && 'intersectionRatio' in window.IntersectionObserverEntry.prototype)",
+      path: require.resolve('intersection-observer/intersection-observer.js'),
+      minify: true,
+    });
+  }
+
+  if (polyfills.webcomponents) {
+    addPolyfillConfig(
+      {
+        name: 'webcomponents',
+        test: "!('attachShadow' in Element.prototype) || !('getRootNode' in Element.prototype)",
+        path: require.resolve('@webcomponents/webcomponentsjs/webcomponents-bundle.js'),
+      },
+      '@webcomponents/webcomponentsjs',
+    );
+
+    // If a browser does not support nomodule attribute, but does support custom elements, we need
+    // to load the custom elements es5 adapter. This is the case for Safari 10.1
+    addPolyfillConfig(
+      {
+        name: 'custom-elements-es5-adapter',
+        test: "!('noModule' in HTMLScriptElement.prototype) && 'getRootNode' in Element.prototype",
+        path: require.resolve('@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js'),
+      },
+      '@webcomponents/webcomponentsjs',
+    );
+  }
+
+  /** @type {PolyfillFile[]} */
+  const polyfillFiles = [];
+  /** @type {PolyfillFile | undefined} */
+  let coreJs;
+
+  for (const polyfillConfig of polyfillConfigs) {
+    if (!polyfillConfig.name || !polyfillConfig.path) {
+      throw new Error(`A polyfill should have a name and a path property.`);
+    }
+
+    const codePath = path.resolve(polyfillConfig.path);
+    if (!codePath || !fs.existsSync(codePath) || !fs.statSync(codePath).isFile()) {
+      throw new Error(`Could not find a file at ${polyfillConfig.path}`);
+    }
+
+    let content = fs.readFileSync(codePath, 'utf-8');
+    /** @type {string} */
+    if (polyfillConfig.minify) {
+      const minifyResult = Terser.minify(content, { sourceMap: false });
+      // @ts-ignore
+      content = minifyResult.code;
+    }
+
+    const filePath = `${path.join(
+      cfg.polyfillsDir || 'polyfills',
+      `${polyfillConfig.name}${polyfills.hash !== false ? `.${createContentHash(content)}` : ''}`,
+    )}.js`;
```

Comment:
This will only work for Linux. To support Windows, just use: `path.join(..).replace('\\', '/')`.

Reply:
I do a `toBrowserPath` later on, but not in all locations. I'll move that logic here.